autogpt_platform/frontend/src/components/DataTable.tsx

```diff
@@ -72,17 +74,15 @@ export default function DataTable({
                   >
                     <Clipboard size={18} />
                   </Button>
-                  {value
-                    .map((i) => {
-                      const text =
-                        typeof i === "object"
-                          ? JSON.stringify(i, null, 2)
-                          : String(i);
-                      return truncateLongData && text.length > maxChars
-                        ? text.slice(0, maxChars) + "..."
-                        : text;
-                    })
-                    .join(", ")}
+                  {value.map((item, index) => (
+                    <React.Fragment key={index}>
+                      <ContentRenderer
```

Comment:
Can this inherit from react fragment so it can take key and friends

Reply:
It already is, but there is `index < value.length - 1 && ", "}` below it, hence the fragment still needed.